xml/System.Runtime.Intrinsics/Vector256`1.xml

```diff
@@ -37,8 +37,8 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <typeparam name="T">To be added.</typeparam>
-    <summary>To be added.</summary>
+    <typeparam name="T">The vector type. <c>T</c> can be any primitive numeric type.</typeparam>
```

Comment:
```suggestion
    <typeparam name="T">The vector type. <typeparam name="T"> can be any primitive numeric type.</typeparam>
```

Reply:
I assume you mean "typeparamref" here too, right? And below.

Reply:
Also, these should have a slash at the end, right? Like `<typeparamref name="T" />`

Reply:
Oh yes, sorry about that.

Reply:
I updated the suggestions.

Reply:
> Also, these should have a slash at the end, right? Like `<typeparamref name="T" />`

Yes, good catch.